packages/fluentui/react-northstar-fela-renderer/src/createFelaRenderer.tsx

```diff
@@ -92,7 +92,7 @@ const rendererConfig = {
   ],
 };
 
-export const createFelaRenderer: CreateRenderer = target => {
+export const createFelaRenderer: CreateRenderer = () => {
```

Comment:
should the `CreateRenderer` type be updated to stop accepting a `target` as an arg ?

Reply:
There should be a bigger refactor if we will decide to implement compat mode with Griffel. That refactor will remove that WeakMap and resolve the issue from a different angle.

Reply:
Is this that big of a refactor ? There should only be a couple of `CreateRenderer` functions in the library, and leaving the `target` parameter there is just going to make this easier to repeat, since looking at the types you are _**encouraged**_ to pass a target in there at creation time.

Reply:
> Is this that big of a refactor ? There should only be a couple of `CreateRenderer` functions in the library, and leaving the `target` parameter there is just going to make this easier to repeat, since looking at the types you are _**encouraged**_ to pass a target in there at creation time.

Yes. Emotion renderer has different implementation, https://github.com/microsoft/fluentui/blob/165a46a8e5fbbe4ac8356666a8df6ab95a09ceb7/packages/fluentui/react-northstar-emotion-renderer/src/createEmotionRenderer.tsx#L35

If you're interested, check #20611 and changes in [mergeProviderContexts.ts](https://github.com/microsoft/fluentui/pull/20611/files#diff-271da4e04c66796852e2943fe9c81f59528cfce13d4dca4b38887a1b980be5df). It aligns implementation with v9 and removes the need in Weakmap. I would rather do that change than try to remove `target` from arguments.